sunbird-platform-core/common-util/src/main/java/org/sunbird/common/request/UserRequestValidator.java

```diff
@@ -978,4 +978,20 @@ public void validateUserMergeRequest(Request request) {
               ERROR_CODE);
     }
   }
+
+  public void validateCertValidationRequest(Request request) {
+    if(StringUtils.isBlank((String) request.getRequest().get(JsonKey.CERT_ID))) {
+      throw new ProjectCommonException(
```

Comment:
create a common method that will take ErrorCode and then internally throw ERROR
-- createClientError (ResponseCode responseCode.)
throw new ProjectCommonException(
              ResponseCode.certIdRequired.getErrorCode(),
              ResponseCode.certIdRequired.getErrorMessage(),
              ERROR_CODE);

Reply:
done , created a new method.